frontend/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,167 @@
+//  'sender_zip_code', 'sender_race', 'sender_age', 'sender_gender',
+//  'receiver_id', 'receiver_zip_code', 'receiver_race', 'receiver_age',
+//  'receiver_gender', 'predicted_fraud', 'is_fraud'
+
+interface Column {
+  id: 'transaction_id' | 'transaction_amount' | 'transaction_date' | 'sender_id' |
+       'sender_zip_code' | 'sender_race' | 'sender_age' | 'sender_gender' |
+       'receiver_id' | 'receiver_zip_code' | 'receiver_race' | 'receiver_age' |
+       'receiver_gender' | 'predicted_fraud' | 'is_fraud';
+  label: string;
+  minWidth?: 150;
+  align?: 'right';
+  format?: (value: number) => string;
+}
+
+const columns: readonly Column[] = [
+    { id: 'transaction_id', label: 'transaction_id' },
+    { id: 'transaction_amount', label: 'transaction_amount', align: 'right' },
+    { id: 'transaction_date', label: 'transaction_date' },
+    { id: 'sender_id', label: 'sender_id' },
+    { id: 'sender_zip_code', label: 'sender_zip_code' },
+    { id: 'sender_race', label: 'sender_race' },
+    { id: 'sender_age', label: 'sender_age', align: 'right' },
+    { id: 'sender_gender', label: 'sender_gender' },
+    { id: 'receiver_id', label: 'receiver_id' },
+    { id: 'receiver_zip_code', label: 'receiver_zip_code' },
+    { id: 'receiver_race', label: 'receiver_race' },
+    { id: 'receiver_age', label: 'receiver_age', align: 'right' },
+    { id: 'receiver_gender', label: 'receiver_gender' },
+    { id: 'predicted_fraud', label: 'predicted_fraud' },
+    { id: 'is_fraud', label: 'is_fraud' },
+];
+
+interface Data {
+    transaction_id: string,
+    transaction_amount: number,
+    transaction_date: string,
+    sender_id: number,
+    sender_zip_code: number,
+    sender_race: string,
+    sender_age: number,
+    sender_gender: string,
+    receiver_id: string,
+    receiver_zip_code: number,
+    receiver_race: string,
+    receiver_age: number,
+    receiver_gender: string,
+    predicted_fraud: number,
+    is_fraud: number
+}
+
+function createData(
+    transaction_id: string,
+    transaction_amount: number,
+    transaction_date: string,
+    sender_id: number,
+    sender_zip_code: number,
+    sender_race: string,
+    sender_age: number,
+    sender_gender: string,
+    receiver_id: string,
+    receiver_zip_code: number,
+    receiver_race: string,
+    receiver_age: number,
+    receiver_gender: string,
+    predicted_fraud: number,
+    is_fraud: number
+  ): Data {
+    return {
+      transaction_id,
+      transaction_amount,
+      transaction_date,
+      sender_id,
+      sender_zip_code,
+      sender_race,
+      sender_age,
+      sender_gender,
+      receiver_id,
+      receiver_zip_code,
+      receiver_race,
+      receiver_age,
+      receiver_gender,
+      predicted_fraud,
+      is_fraud,
+    };
+}
+
+const rows = [
+  createData('TX_0_1', 6087.48, '2023-07-07 13:59:48.275141', 0, 56126, 'Black', 39, 'Man', 'Rec_8564', 93889, 'Hispanic', 52, 'Woman', 0, 1),
```

Comment:
small nitpick: for the gender part maybe switch example to 'Female' and 'Male'

Reply:
I wanted to leave the option for future datasets to have "Man", "Woman", "Nonbinary" etc. so it aligns more with gender rather than sex, and the converter class filters for unique values so it can accommodate more than two options (if there are any)